addon/components/o-s-s/select.ts

```diff
@@ -66,6 +76,10 @@ export default class OSSSelect extends Component<OSSSelectArgs> {
     return classes.join(' ');
   }
 
+  noop(event: Event): void {
+    event.stopPropagation();
+  }
```

Comment:
Same here

Reply:
it's because when the dropdown is portaled into the body, clicking in it (not the options but like empty state or corners) actually counts as a click outside 😅  this catches it